AfrikaBurn/BurnElementsViewController.swift

```diff
@@ -218,6 +259,35 @@ class BurnElementsViewModel {
         return elements.results[index]
     }
     
+    private func applySearchTextFilter(searchText : String) -> CurrentElements{
+        
+        let newElements: CurrentElements
+        
+        // if it's an empty search then just show previous filter
+        if (searchText.isEmpty){
+            switch displayMode {
+            case .default:
+                newElements = .normal(applyFilter(activeFilter, to: self.allElements))
+            case .favorites:
+                newElements = .favorites(applyFilter(activeFilter, to: persistentStore.favorites()))
+            }
+            return newElements
+        }
+        
+        
+        let searchedElements = self.allElements.filter("name CONTAINS[c] '\(searchText)' OR longBlurb CONTAINS[c] '\(searchText)'")
```

Comment:
Yeah i thought about this. Should search only search your subset of filtered results or all results? 
I tend to lean towards always search all results because the number of items is so small and if you are looking for a specific item but you have the wrong filter set it won't show.

Don't feel super strongly about it but thats just my thoughts. :D